src/overmap.cpp

```diff
@@ -5115,11 +5124,32 @@ spawns happen at... <cue Clue music>
 20:56 <kevingranade>: game:pawn_mon() in game.cpp:7380*/
 void overmap::place_cities()
 {
+    int city_size_scalar = 0; // used to increase city size as we move East and South.
+    int city_space_scalar =
+        0; // used to space cities out as we go more into the West and the Appalachians.
```

Comment:
This looks a little better.
```suggestion
    // used to space cities out as we go more into the West and the Appalachians.
    int city_space_scalar = 0;
```